src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -344,27 +327,27 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
               }
             >
               <MenuItem sx={{ color: "gray" }}>{ strings.draftEditorScreen.editor.visibility.selectField }</MenuItem>
-              <MenuItem value="kylla">{ strings.generic.yes }</MenuItem>
-              <MenuItem value="ei">{ strings.generic.no }</MenuItem>
+              <MenuItem value="yes">{ strings.generic.yes }</MenuItem>
+              <MenuItem value="no">{ strings.generic.no }</MenuItem>
             </Select>
```

Comment:
Change this to get the option values from form fields options. Also move upper selects onChange method to its own handler to keep code cleaner.